sources/updater/updog/src/main.rs

```diff
@@ -459,11 +494,11 @@ fn main_inner() -> Result<()> {
                 &manifest,
                 &current_release.version_id,
                 &variant,
-                arguments.force_version,
+                get_target_version(&config, &arguments.force_version)?,
             )
             .context(error::UpdateNotAvailable)?;
 
-            if !arguments.ignore_waves {
+            if !(arguments.ignore_waves || config.ignore_waves) {
```

Comment:
We're doing this in multiple places - can we set a single `ignore_waves` variable using this logic, and then be sure we're not referencing these properties elsewhere anymore?  I don't want to miss changing one of them in the future...